include/lib/cpus/aarch64/dsu_arch.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2018, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef DSU_ARCH_H
+#define DSU_ARCH_H
```

Comment:
minor: strictly speaking, the DSU is mircoarchitecture, not architecture, which might affect this filename

Reply:
I will rename that to dsu_def.h